docs/implementation-status.md

```diff
@@ -198,11 +197,11 @@ in the following sections.
 
 | Procedure | Status | Notes |
 |-----------|--------|-------|
-| `prif_co_broadcast` | ... | ... |
-| `prif_co_max`       | ... | ... |
-| `prif_co_min`       | ... | ... |
-| `prif_co_sum`       | ... | ... |
-| `prif_co_reduce`    | ... | ... |
+| `prif_co_broadcast` | *partial* | no support for derived types with `allocatable` components |
+| `prif_co_max`       | *partial* | only supports 32-bit and 64-bit numeric types |
+| `prif_co_min`       | no        |  |
+| `prif_co_sum`       | no        |  |
```

Comment:
Why were these two changed from the suggestion? 

I believe all three built-in reduction operations (`co_{min,max,sum}`) are currently in the same state, ie they are believed to be working with gfortran and 32/64-bit numeric datatypes. 

These all have some known problems (e.g. issue #132, which is really a PRIF spec issue).  `co_{min,max}` have some support for Fortran strings but it currently relies on undefined behavior of gfortran. None of these work on flang yet, but we're not publicly claiming flang support for anything yet.

However despite all that I think I'd still characterize all three the same way:

```suggestion
| `prif_co_min`       | *partial* | only supports 32-bit and 64-bit numeric types |
| `prif_co_sum`       | *partial* | only supports 32-bit and 64-bit numeric types |
```

Reply:
Re-reading the prior comment thread I think I understand the miscommunication. 

The concerns with `prif_co_{min,max}` are almost entirely regarding the implementation of character type arguments. IMO 32/64-bit numeric datatypes should be considered working.

Reply:
@bonachea Thanks, yes I just misunderstood your earlier comment. Thank you for catching that are for the code suggestion.